recipes/net-snmp/all/conanfile.py

```diff
@@ -0,0 +1,235 @@
+import functools
+import os
+import stat
+from contextlib import contextmanager
+
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.43.0"
```

Comment:
```suggestion
required_conan_version = ">=1.44.0"
```
tool_requires was introduced in 1.44.0

Reply:
Thanks. I will amend and force push.